.github/actions/on_device_tests/action.yaml

```diff
@@ -57,18 +57,21 @@ runs:
         GITHUB_PR_HEAD_USER_ID: ${{ github.event.pull_request.head.user.id }}
         GITHUB_COMMIT_AUTHOR_USERNAME: ${{ github.event.commits[0].author.username }}
         GITHUB_COMMIT_AUTHOR_EMAIL: ${{ github.event.commits[0].author.email }}
+        GITHUB_PR_NUMBER: ${{ github.event.pull_request.number }}
+        GITHUB_RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
       run: |
         set -uxe
         python3 tools/on_device_tests_gateway_client.py \
           --token ${GITHUB_TOKEN} \
-          --change_id "${GITHUB_PR_HEAD_SHA:-$GITHUB_SHA}" \
+          --change_id ${GITHUB_PR_NUMBER:-postsubmit} \
           trigger \
           --test_type ${{ env.TEST_TYPE }} \
           --platform ${{ matrix.target_platform }} \
           --config ${{ matrix.config }} \
           --tag cobalt_github_${GITHUB_EVENT_NAME} \
           --builder_name github_${{ matrix.platform }}_tests \
           --build_number ${GITHUB_RUN_NUMBER} \
+          --builder_url GITHUB_WORKFLOW_RUN \
```

Comment:
Just noticed that I forgot to wrap this in curlies. Will fix.

Reply:
this should be ${GITHUB_RUN_URL} not ${GITHUB_WORKFLOW_RUN}, right?

Reply:
Done